earthaccess/search.py

```diff
@@ -79,11 +77,10 @@ def get_results(
 
 
 class DataCollections(CollectionQuery):
-    """
-    ???+ Info
-        The DataCollection class queries against
-        https://cmr.earthdata.nasa.gov/search/collections.umm_json,
-        the response has to be in umm_json to use the result classes.
+    """???+ Info
+    The DataCollection class queries against
+    https://cmr.earthdata.nasa.gov/search/collections.umm_json,
+    the response has to be in umm_json to use the result classes.
```

Comment:
Looks like the linter messed up the indentation on this info box , so now it renders as empty (here: https://earthaccess--580.org.readthedocs.build/en/580/user-reference/collections/collections-query/)! :(

I think this happened because there's no subject line in the docstring like there should be. I think we need to add that. A placeholder would be OK, since there's nothing there right now. Are you OK with adding `Placeholder.` as the first line for docstrings missing one?